### PR TITLE
#886 - Share on Twitter shortcode

### PR DESCRIPTION
**Pull request by kevincrafts**

### CODE REVIEW

File: modules/custom/cu_share/cu_share.module [hunk 1]
```diff
@@ -537,3 +537,55 @@ function cu_share_flush_caches() {
...
   }
 }
+
+function cu_share_shortcode_info() {
+  $shortcodes = array();
+  $shortcodes['share_on_twitter'] = array(
+    'title' => t('Share On Twitter'),
+    'description' => t('Create a link and quote for readers to share on Twitter.'),
+    'process callback' => 'cu_share_share_on_twitter',
+    'tips callback' => 'cu_share_share_on_twitter_tips',
```
Comment (alexfinnarn) [inline modules/custom/cu_share/cu_share.module]: The tips callback doesn't exist so I'm not sure how to use this shortcode.

**Comment by cathysnider** [comment]
https://www-dev.colorado.edu/feature/886/
Verified: This module is turned on when the Social Media bundle is turned on.

**Comment by ekellagher** [comment]
Web Central Documentation 
https://www.colorado.edu/webcentral/tutorial/v2/twitter-share-shortcode

File: modules/custom/cu_share/cu_share.module [hunk 2]
```diff
@@ -537,3 +537,69 @@ function cu_share_flush_caches() {
...
+function cu_share_share_on_twitter($attrs, $content = NULL) {
+  extract(shortcode_attrs(array(
+  ), $attrs));
+  // Character count - 140, url, ellipsis
+  $character_count = 140;
+  $content = strip_tags($content);
+  if (arg(0) == 'node' && is_numeric(arg(1))) {
+    // Get the nid
+    $nid = arg(1);
+    $current_url = url('node/' . $nid, array('absolute' => TRUE, 'alias' => TRUE));
```
Comment (alexfinnarn) [inline modules/custom/cu_share/cu_share.module]: I think the alias key should be removed. Sending URLs with the alias makes more sense to me than including the system path.

**Pull request merged**